### PR TITLE
[NU-1455] Add missing e2e tests for tests from file

### PR DESCRIPTION
**Pull request by lciolecki**
## Describe your changes
In my last PR: https://github.com/TouK/nussknacker/pull/5786 I fixed the problem with the broken encoding mechanism (with e.g. Avro) when we run test from file on the Flink engine. But I didn't add tests e2e for this fix. 

This PR provides e2e tests (using Kafka, SR as docker services) in Cypress.

## Checklist before merge
- [x] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [x] Code is cleaned from temporary changes and commented out lines
- [x] Parts of the code that are not easy to understand are documented in the code
- [x] Changes are covered by automated tests
- [x] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [x] Verify that PR will be squashed during merge

### CODE REVIEW

File: .run/NussknackerApp.run.xml [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-      <pattern>
-        <option name="PATTERN" value="pl.touk.nussknacker.ui.*" />
-        <option name="ENABLED" value="true" />
-      </pattern>
-    </extension>
-    <method v="2">
-      <option name="Make" enabled="true" />
-    </method>
-  </configuration>
+    <configuration default="false" name="NussknackerApp" type="Application" factoryName="Application">
```
Comment (lciolecki) [inline .run/NussknackerApp.run.xml:2]: I don't know why IDEA has changed this..

**Review by Dzuming — APPROVED** [review]
Frontend changes LGTM

**Review by mslabek — COMMENTED** [review]
Small suggestions/questions. Very valuable change - will enable us to automate a lot more screenshots for documentation and have a more featureful light FE dev setup.

**Review by arkadius — APPROVED** [review]
Nice!

File: e2e-tests/docker-compose.yml [hunk 2]
```diff
@@ -19,28 +17,21 @@ services:
 
   kafka:
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply e2e-tests/docker-compose.yml:18]: In this exmaple we use just kafka, I don't want to change it.

File: .run/NussknackerApp.run.xml (hunk 1 above)
Comment (lciolecki) [reply .run/NussknackerApp.run.xml:2]: Hmm, I'll prepare a separate PR with editorconfig for it..

**Review by mslabek — APPROVED** [review]

**Pull request merged**